esp-hal/CHANGELOG.md

```diff
@@ -65,6 +65,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The `PeripheralInput` and `PeripheralOutput` traits are now sealed (#2690)
 - `esp_hal::sync::Lock` has been renamed to RawMutex (#2684)
 - Updated `esp-pacs` with support for Wi-Fi on the ESP32 and made the peripheral non virtual
+- I2C: Prefer compile-time checks over runtime checks where possible, prefer a fallible API over panics. (#2831)
```

Comment:
I'd prefer not copying the API guidelines into the changelog. Describe what you did instead (replace potential panic with error).

I'm thinking maybe this can go under `Fixed`, also, but I'm not sure.

Reply:
Meeh, `Changed` feels more suitable for me... 

But if you insist, I'll move that 😄